packages/babel-types/test/fields.js

```diff
@@ -72,20 +40,15 @@ const ignoredFields = {
       }),
 };
 
-function isEmpty(obj) {
-  for (const key in obj) return false;
-  return true;
-}
-
 describe("NODE_FIELDS contains all fields, and the visitor order is correct, in", function () {
   const reportedVisitorOrders = new Set();
 
   it.each(files)("%s", async file => {
-    const ast = await readJson(path.resolve(packages, file));
+    const ast = JSON.parse(readFileSync(file, "utf8"));
     if (ast.type === "File" && ast.errors && ast.errors.length) return;
     t[`assert${ast.type}`](ast);
     const missingFields = {};
-    traverse(ast, node => {
+    t.traverseFast(ast, node => {
```

Comment:
The difference between traverse and traverseFast is that the second one only visits known fields, but it's fine because we check for extra fields _before_ recursing.

Reply:
Oh thank you, I realize the new behavior no longer iterates over the children of ignored fields, if any reviewer thinks the old behavior is better I can revert that behavior.

Reply:
I think the new behavior is fine.